crane-cloud/frontend
# Improve Responsiveness at the frontend

# Description

The front-end on small screens does not appear fine, this PR try to improve the responsiveness, it should be checked throughly just in case of un seen broken css 

## Type of change


## Trello Ticket ID

https://trello.com/c/BiCI6tKQ

## How Can This Been Tested?

pull the branch and test it on small phone screens and also on the big ones to see if no change has been made

## Checklist:


# Screenshots
![image](https://user-images.githubusercontent.com/69305400/157007277-34bb7974-6c98-4020-9d81-632eae933e61.png)
![image](https://user-images.githubusercontent.com/69305400/157007439-a2702d3e-a679-467a-ab80-1ee7f3699aa5.png)
